apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -60,18 +70,19 @@ if ( window.starterPageTemplatesConfig ) {
 					className="page-template-modal"
 				>
 					<div className="page-template-modal__inner">
-						<div className="page-template-modal__intro">
-							<p>Pick a Template that matches the purpose of your page.</p>
-							<p>You can customise each Template to meet your needs.</p>
-						</div>
 						<form className="page-template-modal__form">
 							<fieldset className="page-template-modal__list">
+								<legend className="page-template-modal__intro">
+									<p>Pick a Template that matches the purpose of your page.</p>
+									<p>You can customise each Template to meet your needs.</p>
+								</legend>
 								<TemplateSelectorControl
 									label="Template"
 									templates={ Object.values( verticalTemplates ).map( template => ( {
 										label: template.title,
 										value: template.slug,
-										preview: template.preview,
+										preview: template.preview && ( template.preview.src || '' ),
+										previewAlt: template.preview && ( template.preview.alt || '' ),
```

Comment:
Could we simplify that to `template.preview` and `template.previewAlt`?

Reply:
Yes we can now. Done